code/modules/food_and_drinks/restaurant/generic_venues.dm

```diff
@@ -6,8 +6,14 @@
 	max_time_between_visitor = 100 SECONDS
 
 
+/datum/venue/restaurant/New()
+	. = ..()
+
+	// Moths only order food, and so should only be a customer for chefs.
+	customer_types[/datum/customer_data/moth] = 1
```

Comment:
I don't like this runtime addition of customers. Just set the list in its default variables.

Reply:
In the same sense, its weird to look for the spawn list of venues and it being located in two places.

Reply:
In the same sense, its weird to look for the spawn list of venues and it being located in two places.